app/models/katello/concerns/user_extensions.rb

```diff
@@ -9,7 +9,7 @@ module UserExtensions
         include Util::ThreadSession::UserModel
 
         has_many :task_statuses, :dependent => :destroy, :class_name => "Katello::TaskStatus"
-        has_many :activation_keys, :dependent => :destroy, :class_name => "Katello::ActivationKey"
+        has_many :activation_keys, :class_name => "Katello::ActivationKey"
```

Comment:
ugg I can't believe that was there.  shouldn't we switch it to `:dependent => :nullify` though?

otherwise we might hit a foreign key constraint (assuming there is one, which there should be)